docs/modules/ROOT/pages/getting-started/installation.adoc

```diff
@@ -0,0 +1,62 @@
+= Installation
+
+ifeval::["{page-component-display-version}" == "main"]
+:download-path: https://github.com/kubearchive/kubearchive/releases/latest/download
+endif::[]
+ifeval::["{page-component-display-version}" != "main"]
+:download-path: https://github.com/kubearchive/kubearchive/releases/download/{page-component-display-version}
+endif::[]
+
+This document explains how to install and configure KubeArchive in your Kubernetes cluster.
+
+== Prerequisites
+
+* A PostgreSQL instance
+* CertManager is installed on the Kubernetes cluster (+v1.9.1)
+* Knative Eventing is installed on the Kubernetes cluster (+v1.15.0)
+
+== Prepare the Database
+
+. Create ada user and a password for KubeArchive to use.
+. Create a database for KubeArchive, with admin permissions granted to the user you just created.
+. As the user, execute the `ddl-resource.sql` we provide for initializing the database, located at:
```

Comment:
This should all be *in* the SQL file. There's no reason for the user to do this.

> Create ada user

I also think there's a typo here.

Reply:
Fixed the typo. I created #381 to discuss this.